Add colors to the tui monitor view

wjhoward opened the pull request:
I've really enjoyed using Oha, thank you for your hard work. 

This PR adds color to the main monitor view, using the Bang Wong color blind friendly palette. This can be optionally disabled using the `--disable-color` flag.

I did consider adding terminal capable color checks, e.g. checking `$COLORTERM` / `$TERM` environment variables, although this doesn't work for all platforms (e.g. Windows Powershell). All modern terminals will support colors, and I've tested this on MacOs - Iterm2 and Windows 10 Powershell ( I don't have an Ubuntu Gnome terminal to hand ).

<img width="1486" alt="image" src="https://user-images.githubusercontent.com/61805721/189524512-d65171b2-71d6-489e-af1d-66d6b8b27b49.png">

I did consider adding color ranges to the printed latency output, as per issue: https://github.com/hatoo/oha/issues/64 but I hesitated, as latency ranges can vary considerably per application.

The README etc would need to be updated, but wanted to get your thoughts on the change first.

hatoo commented:
Great👍 
Please go ahead.
Don't forget to update `CHANGELOG.md`!

wjhoward commented:
Great, thanks! Do you want me to update the `demo.gif` also?

hatoo commented:
Yeah, if you update `demo.gif` also, it's very helpful.

wjhoward commented:
> Yeah, if you update `demo.gif` also, it's very helpful.

Added a new `demo.gif`, the dimensions are lower than the original, but this seems largely due to the fact I tried to keep the amount of detail the same in the gif using a small terminal window. If I draw a bigger window (with higher dimensions) I get much more detail, e.g. many more bars. It looks pretty good in the README, e.g. https://github.com/wjhoward/oha/tree/add-tui-colors

wjhoward commented:
Let me know if you want me to make any changes or add anything else before considering merging this? Thanks!

hatoo commented on CHANGELOG.md:
```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+# 0.5.5 (2022-09-12)
```
Please remove this line.
I'll add this line when we release a new version

hatoo commented:
Sorry, I thought I added a review yesterday but it was `pending` state.
I've added reviews now, could you see my reviews?

wjhoward commented:
> Sorry, I thought I added a review yesterday but it was `pending` state. I've added reviews now, could you see my reviews?

No problem! I can see the review now, will make the required changes shortly. Thanks

hatoo commented:
Thanks! merged.

Vanav commented:
I don't see colors in tui for some reason: 
![image](https://user-images.githubusercontent.com/761333/217436027-44a82963-121c-4339-8882-c22953ffb49f.png)
Also notice that progress bar is invisible in "color" mode.

oha v0.5.6 (amd64 build from Releases page), Ubuntu 22 / Debian 11, TERM=xterm-256color.
Ternimal has no issues with colors, even oha shows some: 
![image](https://user-images.githubusercontent.com/761333/217436439-f34b08d5-bf93-4e2b-9e33-a4e6bb5c3979.png)
